contributor/outline.md

```diff
@@ -11,21 +11,22 @@ Outline for this section inspired by the [Trusted Committer introduction video].
 
 ## Becoming an InnerSource Contributor
 
-  - A contributing mindset starts with a shared use mindset.
-    - When you have technical needs that are outside of the domain of your project,
+  - A contributing mindset starts with a sharing mindset.
+    - When you have technical needs that are orthoganal of the domain of your project,
     look around to see if there is a shared solution that you can use.
     - Briefly enumerate the benefits of using shared solutions.
   - Using shared solutions puts you in a position to become a _Contributor_.
     - There will be times when you wish that the shared solution has some new feature.
-    - Don't _Wait it Out_ or _Work Around_, spend your time and energy making the change directly in the shared project.
+    - Don't _Wait it Out_ or _Work Around_; spend your time and energy making the change directly in the shared project.
     - Briefly enumerate the benefits of contributing.
     This should be taken from the **Benefits** segment.
-    - You may need to spend a bit of time explaining these benefits to your management hierarchy in order to have their support to spend your time contributing.
-    - An InnerSource _Contributor_ knows that it doesn't matter where the code goes as long as they get the needed functionality.
-  - Contributing applies to all levels of project managements, not just code.
+    - You may need to spend some time explaining these benefits to your management hierarchy in order for their support for the time you spend contributing.
+    - An InnerSource _Contributor_ knows that it doesn't matter where the code goes as it's in a well-run project that gives them the functionality they need.
+  - The scope of contributions goes beyond "just code".
     - Opening issues.
     - Improving documention.
     - Fixing builds.
+    - A test case that shows a bug in the existing code base.
```

Comment:
:D Extended this thing here - we all may have seen test cases that either are flaky or incorrect, i.e. don't test what they should.
```suggestion
    - Creating or fixing test cases that show a bug in the existing code base.
```